server/api-manager.js

```diff
@@ -35,32 +39,895 @@ class MindbodyQueries {
     );
     this.requestNum++;
     if (!this.atLimit()) {
-      return req.makeRequest();
+      return request.makeRequest();
     }
     return Promise.reject(Error("Request limit reached"));
   }
 
-  //gets clients
-  //returns a promise that will eventually resolve
-  getClients() {
-    var req = new MindbodyRequest(
-      URL_CLIENTS,
+  /**@returns Activation code. Requires mindbody to approve you for live customer data */
+  getActivationCode() {
+    var request = new MindbodyRequest(
+      URL_SITE + "activationcode",
       APIKEY,
       SITEID,
       "GET",
       ""
     );
-    req.addAuth(this.authToken);
+    return this.decorateAndMake(request);
+  }
+
+  /* eslint-disable *//**
```

Comment:
This isn't a deal breaker but it seems silly to use eslint to enforce some code standards but then eslint-disable them all the time. I'd rather:
 - change our eslint settings to be compatible with the kinds of comments we write
 - or find a way to write comments that are compatible with our eslint settings.

Reply:
Eslint is fairly strict and I am using it to auto-format as well. No amount of me tweaking the settings seemed to really let me put something easily human readable in while maintaining the tooltip formatting, so I went with this. I am open to doing this with some eslint changes, but I was having a little trouble getting it to work.